libs/runtime/ebpf_epoch.c

```diff
@@ -29,6 +29,12 @@
  */
 #define EBPF_NANO_SECONDS_PER_FILETIME_TICK 100
 
+#define EBPF_ASSERT_AND_FAIL(X)                   \
+    if (!(X)) {                                   \
+        ebpf_assert(!#X);                         \
+        __fastfail(FAST_FAIL_CORRUPT_LIST_ENTRY); \
```

Comment:
should ebpf_assert do tis internally?   Why or why not?
The name EBPF_ASSERT_AND_FAIL is confusing since I would have thought that ebpf_assert() would fail.
Also, why "CORRUPT_LIST_ENTRY"?  Should that be an input param?